coordinator/static/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Game Node Visualization</title>
+    <script src="scripts/htmx.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="stylesheets/node.css">
+</head>
+<body>
+    <div class="node" hx-get="/nodes" hx-trigger="every 3s"></div>
```

Comment:
Polls for new game nodes every three seconds. In case you don't know about HTMX, this eventually gets compiled to javascript in the browser and just updates this div if there are changes. I chose HTMX so that we wouldn't have to write any javascript :)